contracts/schemes/GenericSchemeMultiCall.sol

```diff
@@ -0,0 +1,197 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "@daostack/infra/contracts/votingMachines/IntVoteInterface.sol";
+import "@daostack/infra/contracts/votingMachines/ProposalExecuteInterface.sol";
+import "../votingMachines/VotingMachineCallbacks.sol";
+
+/**
+ * @title GenericSchemeMultiCall.
+ * @dev  A scheme for proposing and executing calls to multiple arbitrary function
+ * on one or multiple contracts on behalf of the organization avatar.
+ */
+contract GenericSchemeMultiCall is VotingMachineCallbacks, ProposalExecuteInterface {
+
+    // Details of a voting proposal:
+    struct MultiCallProposal {
+        address[] contractsToCall;
+        bytes[] callData;
+        uint256[] value;
+        bool exist;
+        bool passed;
+    }
+
+    mapping(bytes32=>MultiCallProposal) public proposals;
+
+    IntVoteInterface public votingMachine;
+    bytes32 public voteParams;
+    mapping(address=>bool) internal contractWhitelist;
```

Comment:
how about make it public ?

Reply:
@orenyodfat whitelistedContracts is public, having both public would probably confuse